README.md

```diff
@@ -5,12 +5,12 @@
   </a>
 </p>
 
-<h4 align="center">
-Gear is a Substrate-based smart-contract platform allowing anyone to run dApp in a few minutes.
-</h4>
+<h3 align="center">
+Gear Protocol is a Substrate-based platform for developers allowing anyone to run dApp in a few minutes.
```

Comment:
".. protocol is .. platform" sounds weird

Reply:
Gear Protocol is a Substrate-based solution for developers, enabling anyone to run a dApp in just a few minutes.